projects/plugins/jetpack/modules/custom-css/custom-css.php

```diff
@@ -998,6 +1007,62 @@ public static function jetpack_content_width( $content_width ) {
 		return $content_width;
 	}
 
+	/**
+	 * Update the initial state to include the `replace` option (Start Fresh) in the module data.
+	 */
+	public static function update_initial_state() {
```

Comment:
Since this is only meant to be useful in the Jetpack dashboard, I think we should limit it to that page. We don't want to run a posts query on dashboard pages where it won't be used.

Hooking into `admin_enqueue_scripts` instead of `admin_footer` should give you more control there.